Sources/NetworkProtection/Diagnostics/NetworkProtectionConnectionTester.swift

```diff
@@ -178,8 +169,6 @@ final class NetworkProtectionConnectionTester {
     private func scheduleTimer(testImmediately: Bool) async throws {
         await stopScheduledTimer()
 
-        await timerRunCoordinator.start()
```

Comment:
This was moved to `start` above, so that it's immediately set to true once there's a request to start the connection tester.